freqtrade/configuration.py

```diff
@@ -4,6 +4,7 @@
 
 import json
 import logging
+import os
```

Comment:
using "from os import path" would be more pythonic ... (i know this isn't applied everywhere in the bot .. but i think we should try to apply it for new things)

Reply:
great will do this
thanks